pmd-java/src/test/java/net/sourceforge/pmd/lang/java/rule/design/SignatureDeclareThrowsRuntimeExceptionTest.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.design;
+
+import net.sourceforge.pmd.testframework.PmdRuleTst;
+
+public class SignatureDeclareThrowsRuntimeExceptionTest extends PmdRuleTst {
```

Comment:
This class needs an accompanying XML file, that describes the test cases. The file should live under `pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/design/xml/SignatureDeclareThrowsRuntimeExceptionTest`.
See also https://pmd.github.io/pmd-6.12.0/pmd_userdocs_extending_testing.html